azkaban-common/src/main/java/azkaban/trigger/TriggerManager.java

```diff
@@ -329,13 +299,13 @@ private void onTriggerExpire(Trigger t) throws TriggerManagerException {
       List<TriggerAction> expireActions = t.getExpireActions();
       for (TriggerAction action : expireActions) {
         try {
-          logger.info("Doing expire actions");
+          logger.info("Doing expire actions for "+ action.getDescription() + " for " + t);
```

Comment:
do we prefer logger.info("Doing expire actions for {} for {}", action.getDescription(), t)? it makes code more readable